lib/local.js

```diff
@@ -30,10 +30,9 @@ const readLocalFile = () => new Promise((resolve, reject) => {
  * @return {Promise<String[]>} - The list of local dependencies
  */
 const getLocalDependencies = () => new Promise((resolve, reject) => {
-    readLocalFile().then((file) => {
-        const deps = file.dependencies;
-        return deps ? resolve(deps) : reject('No local dependencies defined. Please define a repository.');
-    }).catch(err => reject(err));
+    readLocalFile()
+        .then(files => resolve(files.dependencies || []))
```

Comment:
But yeah, it makes sense. In the (odd) event a user was validating a local `.zel` file that didn't include any dependencies, the old `reject` would interrupt the `validate` action.